### PR TITLE
Upgrade to Ember 3

### PR DESCRIPTION
**Pull request by elwayman02**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +73,35 @@ Composable components make it easy to form a required module, but sometimes it's
 * `$ember-content-placeholders-line-height` - single line height
 * `$ember-content-placeholders-spacing` - distance between lines
 
-## 🔓 License
```
Comment (acorncom) [inline README.md]: Looks like this icon went missing in the upgrade

Comment (elwayman02) [reply README.md]: Apologies, that should be fixed now! For future reference, I think project maintainers have access to edit PRs now when minor changes like this are needed. Sorry for the delay in getting back to this.

**Comment by elwayman02** [comment]
@acorncom I'm not sure why the build is showing as failed. If you click into the travis builds, it actually shows all tests as succeeding and then the build exits.

**Comment by acorncom** [comment]
@elwayman02 Hmm, weird. I'm sadly not a maintainer on this repo, @michalsnik do you have any thoughts?

**Comment by michalsnik** [comment]
Hey @elwayman02 and @acorncom :)

Thanks for working on this! It looks like `The command "yarn lint:js" exited with 1.`

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
+---
 language: node_js
 node_js:
-  - "7"
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "4"
```
Comment (michalsnik) [inline .travis.yml]: I'd also add latest LTS version of Node, just to be sure :)

File: README.md [hunk 3]
```diff
@@ -66,6 +73,35 @@ Composable components make it easy to form a required module, but sometimes it's
...
 
-See the [LICENSE](LICENSE.md) file for license rights and limitations (MIT).
+Contributing
```
Comment (michalsnik) [inline README.md]: Lets add 🍻 here

**Comment by elwayman02** [comment]
@michalsnik should be good to go now! Thanks for the review :)

**Comment by yohanmishkin** [comment]
Awesome, looking forward to this!

**Comment by RuslanZavacky** [comment]
Yay, waiting for this as well :)

**Comment by elwayman02** [comment]
@michalsnik do you have any remaining concerns I should address before you can merge this PR?

**Review by michalsnik — APPROVED** [review]
Thank you @elwayman02 ! LGTM 🚀

**Comment by michalsnik** [comment]
`v1.0.0` released. Enjoy and sorry for the delay guys :)

**Pull request merged**